server.js

```diff
@@ -1,29 +1,121 @@
 import express from "express";
 import cors from "cors";
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// import avocadoSalesData from "./data/avocado-sales.json";
-// import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
-// import topMusicData from "./data/top-music.json";
-
-// Defines the port the app will run on. Defaults to 8080, but can be overridden
-// when starting the server. Example command to overwrite PORT env variable value:
-// PORT=9000 npm start
+import goldenGlobesData from "./data/golden-globes.json";
+import lionMembers from "./data/team-lions.json";
+import booksData from "./data/books.json";
+
 const port = process.env.PORT || 8080;
 const app = express();
 
-// Add middlewares to enable cors and json body parsing
+
 app.use(cors());
 app.use(express.json());
 
-// Start defining your routes here
+//endpoint 1
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.send(
+  {"Welcome": "Hello there! This page is for educational purpose only",
+    "Endpoints/Routes": [
+    {
+      "/books": " Finding array of books",
+      "/books/title/:title": "Finding book by title",
+      "/members": "Finding array of members of Lion group",
+      "/members/name/:name": "Finding single lion member by name",
+      "/nominations": "Finding array of objects with nominations for the Golden Globes Awards",
+      "/nominations/year/:year": "Finding nominations by year (2010-2019)",
+      "/nominations/year/2010?won=true": "Using query to filter out all nominees that won for a given year"
+    }
+  ]
+  });
+});
+
+//endpoint 2
+app.get("/members", (req, res) => {
+
+  res.status(200).json({
+    data: lionMembers,
+    success: true,
+  });
+});
+
+//endpoint 3 
+app.get("/members/name/:name", (req, res)=> {
+  const { name } = req.params;
+  let memberByName = lionMembers.find(
+    (member) => member.name.toLowerCase()
+    .includes(name.toLocaleLowerCase()));
+    
+  if (!memberByName) {
+    res.status(404).json({
+      data: `${name} not found in the database.`,
+      success: false, 
+    });
+  } else {
+    res.status(200).json({
+      data: memberByName,
+      success: true,
+    });
+  }
+});
+
+//endpoint 4 
+app.get("/nominations", (req, res) => {
+
+  res.status(200).json({
+    data: goldenGlobesData,
+    success: true,
+  });
 });
 
+//endpoint 5 
+app.get("/nominations/year/:year", (req, res)=> {
+
+  const year = req.params.year
+  const showWon = req.query.won
```

Comment:
Whoa! Impressed how you combined both query and params into a single endpoint here. Super cool!